tests/sentry/api/endpoints/test_project_stacktrace_link.py

```diff
@@ -119,8 +132,10 @@ def test_no_configs(self):
             "integrations": [serialized_integration(self.integration)],
         }
 
-    def test_file_not_found_error(self):
+    @mock.patch("sentry.integrations.github.integration.GitHubIntegration.get_blame_for_file")
+    def test_file_not_found_error(self, mock_git_blame):
         """File matches code mapping but it cannot be found in the source repository."""
+        mock_git_blame.return_value = None
```

Comment:
We may not need these changes if the "commitSha" field is flag-protected, here and below